### DISCUSSION TITLE
Community contribution world quality plugin

### DISCUSSION THREAD

**Discussion by OracleOfWisdom** [runelite/runelite#15692] [Ideas]
Im thinking you steal a bit of code from tick tracker addon, throw the results onto a server, apply some logic to it so it favors more accurate results (So it doesn't favor someone that has been on the world for 2s) and allow everyone to see a estimated percentile for how good a server is. Possibly a reliability metric on the data as well?

example:

PERSON A:
Time on server: 5000 ticks
Good: 4500 ticks
50+/-: 400 ticks
100+/-:75 ticks
150+/-:25 ticks

PERSON B:
Time on server: 500 ticks
Good: 200 ticks
50+/-: 200 ticks
100+/-:75 ticks
150+/-:25 ticks

Plugin would probably get data in clumps, say every 10 minutes? (In order to be not as intrusive with data being sent) Then show the accumulated data in a simplified view connected to a world switcher, My first thought is just using / as a separator between each value.

Example display for this data set:
W481    5500 / 600 / 150 / 50 / 85%

**Comment by pwatts6060** [comment]
Pretty sure Jagex have been against large scale collection of world lag data before. I remember someone posting some data and having to remove it. The current world you're on is probably fine as a single person, but not every world collated together. 

Actually it was player count on worlds not lag. Still I'd be surprised if they'd be ok with that being collected as it may be used for abuse and confirming certain actions that cause lag for example.
